taipy/core/data/_abstract_file.py

```diff
@@ -13,7 +13,7 @@
 import shutil
 
 
-class _AbstractFileDataNode(object):
+class _AbstractFileDataNodeMixin(object):
```

Comment:
At the end, it is not abstract and it is not a data node.

```suggestion
class _FileMixin(object):
```

Reply:
On the other hand... _FileMixin is really really vague...
But local to core/data... I'm good with that

Reply:
how about `_FileDataNodeMixin`?